plugin.cpp

```diff
@@ -299,45 +352,43 @@ void parseConfig(ConfigCategory *config, EmailCfg *emailCfg)
 	
 }
 
-void validateConfig(PLUGIN_HANDLE *handle, EmailCfg emailCfg)
+
+/**
+ * Validate count of email address and name pairs 
+ */
+void validateConfig(PLUGIN_HANDLE *handle, EmailCfg *emailCfg)
 {
 	PLUGIN_INFO *info = (PLUGIN_INFO *) handle;
-
-	// Check for complete configuation
-	if (emailCfg.email_to == "" || emailCfg.email_from =="" || emailCfg.server == "" || emailCfg.port == 0)
+	// Check for complete configuration
+	int addressCheckSum = emailCfg->email_to.size() + emailCfg->email_cc.size() + emailCfg->email_bcc.size() ;
+	
+	if ( addressCheckSum == 0 || StringStripWhiteSpacesAll(emailCfg->email_from) =="" || StringStripWhiteSpacesAll(emailCfg->server) == "" || emailCfg->port == 0)
 	{
-		info->isConfigValid;
-		Logger::getLogger()->error("To address, From address SMTP Server , SMTP port can not be blank. Please provide complete configuation");
+		info->isConfigValid = false;
+		Logger::getLogger()->error("To address, From address SMTP Server , SMTP port can not be blank. Please provide complete configuration");
```

Comment:
I think better to check these errors separately and provide a specific error msg.